projects/vdk-control-cli/src/vdk/internal/control/job/job_config.py

```diff
@@ -27,7 +31,22 @@ def __init__(self, data_job_path):
                 "Make sure the file is created "
                 "or double check the data job path is passed correctly.",
             )
-        self._config_ini.read(self._config_file)
+        try:
+            self._config_ini.read(self._config_file)
+        except (configparser.MissingSectionHeaderError, Exception) as e:
```

Comment:
configparser is only used for two objects, might be better to import them specifically but this is a nitpick on my part

Reply:
Did some refactoring. Hopefully, it is clearer now